pages/episodes/[slug].vue

```diff
@@ -53,6 +53,21 @@ const isMobile = computed(() => {
             <div class="grid">
               <div class="col">
                 <div v-if="dataLoaded" class="episode flex">
+                  <Html>
+                    <Head>
+                      <Title>{{ episode.title }} | Radiolab | WNYC Studios</Title>
+                      <Meta name="description" :content="episode.tease" />
+                      <Meta name="og:title" :content="`${episode.title} | Radiolab | WNYC Studios`" />
+                      <Meta name="og:description" :content="episode.tease" />
+                      <Meta name="og:type" content="article" />
+                      <Meta name="og:image" :content="episode['image-main'].url" />
```

Comment:
just pointing to the "episode['image-main'].url" will give you the full/source/biggest file available correct? I think you need to use the template like we have been using, and provide the size we need for the OG/META image:

"episode['image-main'].template.replace('%s/%s/%s/%s', '1200/630/c/80')"